### ISSUE TITLE
Resolve conflict opens external merge tool with wrong file order

### ISSUE THREAD

**Issue by GoogleCodeExporter** [guberm/tortoisegit#2205]
```
What steps will reproduce the problem?
1. Set up TortoiseGit to use Beyond Compare Pro 3 as merge tool
2. Produce a conflict while merging
3. Resolve the conflict

What is the expected output? What do you see instead?
Beyond Compare 3-way merge opens. But the expected order of files in the GUI is 
this: LOCAL, BASE, REMOTE. The actual order is now: BASE, LOCAL, REMOTE. This 
leads to an entirely wrong merging behaviour where all my local changes are 
basically reverted. When I swap the two files manually, everything works 
correctly and the merge is correct.

What version of TortoiseGit and msysgit are you using? On
what operating system?
TortoiseGit 1.8.3.0, msysgit 1.8.1.msysgit.1, on Windows 7 x64

Please provide any additional information below.

```

Original issue reported on code.google.com by `yves.goe...@gmail.com` on 27 May 2014 at 7:24
